homeassistant/components/rfxtrx/__init__.py

```diff
@@ -83,10 +94,76 @@
 _LOGGER = logging.getLogger(__name__)
 DATA_RFXOBJECT = "rfxobject"
 
+
+STANDARD_PLATFORM_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_DEVICES, default={}): {
+            cv.string: vol.Schema(
+                {vol.Optional(CONF_FIRE_EVENT, default=False): cv.boolean}
+            )
+        },
+        vol.Optional(CONF_AUTOMATIC_ADD, default=False): cv.boolean,
+        vol.Optional(
+            CONF_SIGNAL_REPETITIONS, default=DEFAULT_SIGNAL_REPETITIONS
+        ): vol.Coerce(int),
+    }
+)
+
+SENSOR_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_DEVICES, default={}): {
+            cv.string: vol.Schema(
+                {
+                    vol.Optional(CONF_FIRE_EVENT, default=False): cv.boolean,
+                    vol.Optional(CONF_DATA_TYPE, default=[]): vol.All(
+                        cv.ensure_list, [vol.In(DATA_TYPES.keys())]
+                    ),
+                }
+            )
+        },
+        vol.Optional(CONF_AUTOMATIC_ADD, default=False): cv.boolean,
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+BINARY_SENSOR_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_DEVICES, default={}): {
+            cv.string: vol.Schema(
+                {
+                    vol.Optional(CONF_DEVICE_CLASS): DEVICE_CLASSES_SCHEMA,
+                    vol.Optional(CONF_FIRE_EVENT, default=False): cv.boolean,
+                    vol.Optional(CONF_OFF_DELAY): vol.Any(
+                        cv.time_period, cv.positive_timedelta
+                    ),
+                    vol.Optional(CONF_DATA_BITS): cv.positive_int,
+                    vol.Optional(CONF_COMMAND_ON): cv.byte,
+                    vol.Optional(CONF_COMMAND_OFF): cv.byte,
+                }
+            )
+        },
+        vol.Optional(CONF_AUTOMATIC_ADD, default=False): cv.boolean,
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
 BASE_SCHEMA = vol.Schema(
     {
         vol.Optional(CONF_DEBUG, default=False): cv.boolean,
```

Comment:
I just noticed that we're using `ATTR_DEBUG` when accessing the config instead of `CONF_DEBUG`. Same for dummy and device items. We should probably fix that.